File: sdk/src/main/java/org/maibot/sdk/exceptions/ClassNoImplementation.java
package org.maibot.sdk.exceptions;

/**
 * 当请求的类没有任何实现类时抛出此异常</br>
 * When no implementation class is found for the requested class, this exception is thrown.
 */
public class ClassNoImplementation extends UnignorableException {
    public ClassNoImplementation(String message, Object... args) {
        super(message, args);
    }
}

File: sdk/src/main/java/org/maibot/sdk/exceptions/ClassNoImplementation.kt
package org.maibot.sdk.exceptions

/**
 * 当请求的类没有任何实现类时抛出此异常
 * When no implementation class is found for the requested class, this exception is thrown.
 */
class ClassNoImplementation(message: String, vararg args: Any?) : UnignorableException(message, *args)
